src/components/Calendars/ClientScheduleAppointment.tsx

```diff
@@ -0,0 +1,464 @@
+'use client';
+import React, { useState, useEffect, useCallback } from 'react';
+import useSWR from 'swr';
+
+//@ts-ignore
+const fetcher = (...args) => fetch(...args).then((res) => res.json());
+
+export default function ClientScheduleAppointment() {
+    //CHANGE THIS ID TO CATARINA'S
+    const today = new Date().toISOString().substring(0, 10);
+    const { data, isLoading } = useSWR(
```

Comment:
We should use clientFetcher it use useSwr on background

Reply:
ditto